Integrations/FortiGate/FortiGate.py

```diff
@@ -429,6 +467,76 @@ def create_firewall_service_request(service_name, tcp_range, udp_range):
     return response
 
 
+def ban_ip(ip_addresses_array, time_to_expire):
+    uri_suffix = 'monitor/user/banned/add_users/'
+
+    payload = {
+        'ip_addresses': ip_addresses_array
+    }
+    if time_to_expire:
+        time_to_expire_int = convert_arg_to_int(time_to_expire, "expiry")
+        payload["expiry"] = time_to_expire_int
+    response = http_request('POST', uri_suffix, {}, json.dumps(payload))
+    return response
+
+
+def ban_ip_command():
+    ip_addresses_string = demisto.args()["ip_addresses"]
+    ip_addresses_array = ip_addresses_string.split(",")
+    time_to_expire = demisto.args().get("expiry")
+    response = ban_ip(ip_addresses_array, time_to_expire)
+    demisto.results({
+        'Type': entryTypes['note'],
+        'ContentsFormat': formats['json'],
+        'Contents': response,
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': 'IPs {0} banned successfully'.format(ip_addresses_string)
+    })
+
+
+def unban_ip(ip_addresses_array):
+    uri_suffix = 'monitor/user/banned/clear_users/'
+
+    payload = {
+        'ip_addresses': ip_addresses_array
+    }
+    response = http_request('POST', uri_suffix, {}, json.dumps(payload))
+    return response
+
+
+def unban_ip_command():
+    ip_addresses_string = demisto.args()["ip_addresses"]
+    ip_addresses_array = ip_addresses_string.split(",")
+    response = unban_ip(ip_addresses_array)
+    demisto.results({
+        'Type': entryTypes['note'],
+        'ContentsFormat': formats['json'],
+        'Contents': response,
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': 'IPs {0} un-banned successfully'.format(ip_addresses_string)
+    })
+
+
+def get_banned_ips():
+    uri_suffix = 'monitor/user/banned/select/'
+    response = http_request('GET', uri_suffix)
+    return response
+
+
+def get_banned_ips_command():
```

Comment:
what about adding those functions to the execution code (command switch-case)

Reply:
Was written in the integration but forgotten on merge